builds/android/app/src/main/java/org/easyrpg/player/player/EasyRpgPlayerActivity.java

```diff
@@ -83,6 +88,7 @@ protected void onCreate(Bundle savedInstanceState) {
 				getWindow().setFlags(0x01000000, 0x01000000);
 			}
 		} catch (Exception e) {
+            e.printStackTrace();
```

Comment:
One thing to note here, is that you use spaces for indentation in these lines again.
Would be really nice if you could convert them so this file is not "mixed".